about.html

```diff
@@ -197,9 +197,9 @@ <h2 id="contribute">Contribute</h2>
                 <li>If you are comfortable with Node or Rhino, <a href="https://github.com/stubbornella/csslint/tree/master/src/cli">test out the command line version</a>, submit feature requests.</li>
             </ol>
             <h2 id="node">CSS Lint Command Line</h2>
-            <p>There are two options for using CSS Lint on the command line. First, if you have Node.js and <a href="http://npmjs.org">npm</a> installed, you can install CSS Lint easily using the following:</p>
+            <p>There are two options for using CSS Lint on the command line. First, if you have Node.js and <a href="https://npmjs.org">npm</a> installed, you can install CSS Lint easily using the following:</p>
             <pre><code>sudo npm install -g csslint</code></pre>
-            <p>If you prefer to use <a href="http://www.mozilla.org/rhino">Rhino instead</a>, grab <a href="http://github.com/stubbornella/csslint/tree/master/release/csslint-rhino.js">csslint-rhino.js</a>.</p>
+            <p>If you prefer to use <a href="http://www.mozilla.org/rhino">Rhino instead</a>, grab <a href="https://github.com/stubbornella/csslint/tree/master/release/csslint-rhino.js">csslint-rhino.js</a>.</p>
```

Comment:
@coliff , this URL is giving a 404. Please change it to https://github.com/CSSLint/csslint/blob/master/dist/csslint-rhino.js .